clinica/iotools/utils/data_handling.py

```diff
@@ -160,6 +163,9 @@ def create_merge_file(
     col_list.insert(1, col_list.pop(col_list.index("session_id")))
     merged_df = merged_df[col_list]
 
+    tmp = merged_df.select_dtypes(include=[np.number])
+    # Round numeric values in dataframe to 12 floating point values
+    merged_df.loc[:, tmp.columns] = np.round(tmp, 12)
```

Comment:
It seems that we are the one that normalize these scores to between 0-1, I think we can round this to something more reasonable

Reply:
What it would be more reasonable ? 6 decimals, for example ?

Reply:
Depends on the range and order of accuracy of the source values.

Reply:
I suppose that different clinical data (coming from different datasets) has different range of values. Can we deduce an order of accuracy that is common to all the dataset processed by Clinica ?

Reply:
I'm currently investigating the issue again by looking into the code of create_tsv. Do you remember if the rounding issue was with adas_Qx scores or something else?

Reply:
It seems (part of the problem) might come from the files [atlas statistics](https://aramislab.paris.inria.fr/clinica/docs/public/latest/Pipelines/T1_Volume/#t1-volume-volume-based-processing-of-t1-weighted-mr-images-with-spm) tsv files generated by the `t1-volume`pipeline. It might then indeed be relevant to keep the precision that high for these cases as it is generated by a software.

Reply:
Yes, I confirmed that these values come from the atlases used in `t1-volume` and `pet-volume` (column names: `t1-volume_group-ADNIbl_atlas*` and `pet-volume_group-ADNIbl_atlas-*`). Do we keep all these decimal values?

Reply:
After discussion with @nburgos,  6 decimal digits seems to be largely sufficient for these values which are averages. If no objection, we can change to that and merge.

Reply:
Done! 
I'll merge when the CI pass.